docs/website/docs/general-usage/http/rest-client.md

```diff
@@ -590,6 +590,22 @@ for page in paginate("https://api.example.com/posts"):
     print(page)
 ```
 
+
+## Retry
```

Comment:
I think it would be beneficial to repeat the docs here, wdyt?
Or, should we only place a link to the request wrapper docs?

Reply:
we agreed upon that in meeting